Sources/HealthKitOnFHIR/HealthKit Extensions/HKCategorySample+Observation.swift

```diff
@@ -0,0 +1,84 @@
+            valueString = HKCategoryValueAppetiteChanges(rawValue: self.value)?.description
+        case HKCategoryType(.appleStandHour):
+            valueString = HKCategoryValueAppleStandHour(rawValue: self.value)?.description
+        case HKCategoryType(.appleWalkingSteadinessEvent):
+            valueString = HKCategoryValueAppleWalkingSteadinessEvent(rawValue: self.value)?.description
+        case HKCategoryType(.cervicalMucusQuality):
+            valueString = HKCategoryValueCervicalMucusQuality(rawValue: self.value)?.description
+        case HKCategoryType(.contraceptive):
+            valueString = HKCategoryValueContraceptive(rawValue: self.value)?.description
+        case HKCategoryType(.environmentalAudioExposureEvent):
+            valueString = HKCategoryValueEnvironmentalAudioExposureEvent(rawValue: self.value)?.description
+        case HKCategoryType(.headphoneAudioExposureEvent):
+            valueString = HKCategoryValueHeadphoneAudioExposureEvent(rawValue: self.value)?.description
+        case HKCategoryType(.lowCardioFitnessEvent):
+            valueString = HKCategoryValueLowCardioFitnessEvent(rawValue: self.value)?.description
+        case HKCategoryType(.menstrualFlow):
+            valueString = HKCategoryValueMenstrualFlow(rawValue: self.value)?.description
+        case HKCategoryType(.ovulationTestResult):
+            valueString = HKCategoryValueOvulationTestResult(rawValue: self.value)?.description
+        case HKCategoryType(.pregnancyTestResult):
+            valueString = HKCategoryValuePregnancyTestResult(rawValue: self.value)?.description
+        case HKCategoryType(.progesteroneTestResult):
+            valueString = HKCategoryValueProgesteroneTestResult(rawValue: self.value)?.description
+        case HKCategoryType(.sleepAnalysis):
+            valueString = HKCategoryValueSleepAnalysis(rawValue: self.value)?.description
+        case
+            HKCategoryType(.irregularHeartRhythmEvent),
+            HKCategoryType(.lowHeartRateEvent),
+            HKCategoryType(.highHeartRateEvent),
+            HKCategoryType(.mindfulSession),
+            HKCategoryType(.toothbrushingEvent),
+            HKCategoryType(.handwashingEvent),
+            HKCategoryType(.sexualActivity),
+            HKCategoryType(.intermenstrualBleeding),
+            HKCategoryType(.infrequentMenstrualCycles),
+            HKCategoryType(.irregularMenstrualCycles),
+            HKCategoryType(.persistentIntermenstrualBleeding),
+            HKCategoryType(.prolongedMenstrualPeriods),
+            HKCategoryType(.lactation):
+            // Samples of these types do not carry any value,
+            // nor associated metadata, so we use the category
+            // identifier as the value.
+            valueString = categoryTypeString
+        default:
+            throw HealthKitOnFHIRError.notSupported
+        }
+        
+        guard let valueString else { throw HealthKitOnFHIRError.notSupported }
```

Comment:
I would suggest to use line breaks for guard statements to more easily see the throwing expression.
```suggestion
        guard let valueString else {
            throw HealthKitOnFHIRError.notSupported
        }
```